Does not work on Windows because of symlink

The symlink fra node/lib/retire.js to chrome/js/retire.js does not work on Windows, or atleast I couldn't manage to get it working. Any tips?

By default running retire in Windows gives the following error: 

```
c:\code\retire.js\node\lib\retire.js:1
(function (exports, require, module, __filename, __dirname) { ../../chrome/js/
                                                              ^
SyntaxError: Unexpected token .
    at Module._compile (module.js:439:25)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (c:\code\retire.js\node\bin\retire:5:17)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
```

I tried to symlink, but couldn't make it work:

```
-> /c/code/retire.js (master)
$ rm node/lib/retire.js

-> /c/code/retire.js (master)
$ node --eval "require('fs').symlinkSync('chrome/js/retire.js', 'node/lib/retir
e.js', 'file')"

-> /c/code/retire.js (master)
$ node node/bin/retire

module.js:340
    throw err;
          ^
Error: Cannot find module '../lib/retire'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (c:\code\retire.js\node\bin\retire:5:17)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
```

--- a/README.md
+++ b/README.md
@@ -21,3 +21,13 @@ _Description_
 Chrome plugin
 -------------
 _Description_
+
+Windows quirks
+---------------
+Requires NTFS filesystem, manually create the symlink with the following commands as administrator:
+```
+cd node\lib\
+del retire.js
+node --eval "require('fs').symlinkSync('../../chrome/js/retire.js', 'retire.j
+s', 'file')"
+

